merger

```diff
@@ -0,0 +1,20 @@
+def combine_chances(textual_chance: float, numerical_chance: float):
+    # 1. Combine the two numbers into an average value
+    average_value = (textual_chance + numerical_chance) / 2
+
+    # 2. Display the median of the two values
+    median_value = sorted([textual_chance, numerical_chance])[1]  # Since there are only two values, the second one is the median
+
+    # 3. Show the maximum value
```

Comment:
same as display, also comment is not needed, `max_value` is readable as is

Reply:
the same for min_value